CMakeLists.txt

```diff
@@ -54,6 +52,7 @@ if(APPLE)
     set(PRODUCTBUILD_KEYCHAIN_PATH "" CACHE FILEPATH "pkgbuild keychain file")
     if(IOS)
         set(XCODE_PROVISIONING_PROFILE_SPECIFIER "" CACHE STRING "Xcode provisioning profile specifier")
+        set(MOLTEN_VK_SDK "$ENV{VULKAN_SDK}/../MoltenVK" CACHE STRING "MoltenVK directory inside the Vulkan SDK")
```

Comment:
If VULKAN_SDK isn't set then MOLTEN_VK_SDK will end up being set to `../MoltenVK`. It might be better if it is set to an empty string in this case. I also found that if VULKAN_SDK is set incorrectly at the initial configure (using the CMake GUI) then MOLTEN_VK_SDK will be wrong, of course.  If you then fix VULKAN_SDK and configure again, MOLTEN_VK_SDK does not change. I would prefer if it picked up the new value of VULKAN_SDK but I don't know how to do that and I don't know if that is acceptable practice in CMake. Do you know @DDoS?

Reply:
I'm not sure if we could differentiate between `VULKAN_SDK` changing, and the user choosing a different `MOLTEN_VK_SDK` with the current scheme.

What might work is to use a different variable; initialized to `${VULKAN_SDK}/../MoltenVK` if `MOLTEN_VK_SDK` is empty, otherwise initialized to `${MOLTEN_VK_SDK}`. The default value of `MOLTEN_VK_SDK` could then be empty left empty by the user for the default path, and a change in `${VULKAN_SDK}` would change the MoltenVK SDK path.

Reply:
Let's leave it as is.